test/stress_tests.py

```diff
@@ -345,22 +360,92 @@ def bar(i, size):
       value = ray.get(args[i])
       self.assertEqual(value[0], i)
 
-    # Wait for errors from all the nondeterministic tasks.
-    time_left = 100
-    while time_left > 0:
-      errors = ray.error_info()
-      if len(errors) >= num_objects / 2:
-        break
-      time_left -= 0.1
-      time.sleep(0.1)
-
-    # Make sure that enough errors came through.
-    self.assertTrue(len(errors) >= num_objects / 2)
+    def error_check(errors):
+      return len(errors) >= num_objects / 2
```

Comment:
while we're at it, can you print something like

```python
print("len(errors) is {}".format(len(errors)))
```

since this test seems to be failing a lot #356.

If that results in too much printing, maybe we can sleep in large increments than `0.1`..